packages/node_modules/@ciscospark/plugin-conversation/src/conversation.js

```diff
@@ -469,10 +469,16 @@ const Conversation = SparkPlugin.extend({
   /**
    * Creates a ShareActivty for the specified conversation
    * @param {Object} conversation
+   * @param {Object} activity
    * @returns {ShareActivty}
    */
-  makeShare(conversation) {
-    return ShareActivity.create(conversation, null, this.spark);
+  makeShare(conversation, activity) {
```

Comment:
I am not comfortable making that change on the web-client side as it appears the plan was not documented anywhere. At this point it would help if we can just go with the above changes and may be revisit it after the sdk v7 upgrade. Please suggest.